### DISCUSSION TITLE
Running Octoprint in Windows 11

### DISCUSSION THREAD

**Discussion by dorofino** [mriscoc/Ender3V2S1#1391] [General]
Hi everyone,

I’m having trouble running OctoPrint on Windows 11. It doesn’t detect my printer connection or correctly identify the baud rate, even though I’m connected on COM#. I’ve tried matching different baud rates and using auto-detect, but it still won’t connect. Does anyone have advice on getting OctoPrint to work properly in Windows?

![image](https://github.com/user-attachments/assets/1e9df3b5-f6fa-40f8-a146-59b7c41d6f01)

![image](https://github.com/user-attachments/assets/15292b4e-ab70-4a2e-b34c-5d44741835bf)

**Comment by mriscoc** [comment]
https://github.com/mriscoc/Ender3V2S1/wiki/Octoprint
